docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -143,6 +143,9 @@ In this example, a dedicated pool of 32MB is allocated for clustered-cache from
 <7> Configures another cache (`shared-cache-2`) that shares the resource pool (`resource-pool-a`) with `shared-cache-1`.
 <8> Creates fully initialized cache manager with the clustered caches.
 
+NOTE:  When a cache is allocated a block of memory from a shared pool, it is retained forever and would never get
```

Comment:
I would put NOTE in normal case but in bold **Note:**. And only one space after

Reply:
'NOTE' here is asciidoc syntax for admonition para, 'Note' will not have the same effect. Removed one extra space though.

Reply:
Cool. Didn't know that.